test/modules/TestResultCoordinator/Reports/TestResultReportBase.cs

```diff
@@ -8,12 +8,15 @@ namespace TestResultCoordinator.Reports
     /// </summary>
     abstract class TestResultReportBase : ITestResultReport
     {
-        protected TestResultReportBase(string trackingId, string resultType)
+        protected TestResultReportBase(string trackingId, string resultType, string testDescription)
```

Comment:
why put testDescription in 3rd argument, but not 1st?  Please think about consistently between other test reports and TestResultReportBase.

Reply:
Oops. Will fix.